cmd/plugins/balloons/policy/balloons-policy.go

```diff
@@ -1526,7 +1526,13 @@ func (p *balloons) pinCpuMem(c cache.Container, cpus cpuset.CPUSet, mems idset.I
 			c.SetCPUShares(int64(cache.MilliCPUToShares(int64(mCpu))))
 		}
 	}
-	if p.bpoptions.PinMemory == nil || *p.bpoptions.PinMemory {
+	// Start from policy-level PinMemory...
+	pinMemory := p.bpoptions.PinMemory == nil || *p.bpoptions.PinMemory
+	// ...and allow override in balloon-type-level PinMemory
+	if (blnDefPinMemory != nil) {
+		pinMemory = *blnDefPinMemory
+	}
+	if pinMemory {
```

Comment:
I think we should let also libmem know about the new container consuming memory, even if we don't ask libmem to determine the pinning according to its own taste. Otherwise it cannot even in theory determine correctly when a node is in danger of running out of memory.